packages/legend-query-builder/src/components/explorer/QueryBuilderFunctionsExplorerPanel.tsx

```diff
@@ -431,14 +426,21 @@ export const QueryBuilderFunctionsExplorerPanel = observer(
       setShowDependencyFuncions(!showDependencyFuncions);
     };
 
+    useEffect(() => {
+      queryFunctionsState.initializeTreeData();
```

Comment:
would this be re-run if someone closes and reopens the function explorer tab?
does `initializeTreeData` need logic to see if the tree has already been created?
We should only do this once pre `QueryBuilderState`